src/test/java/com/mekomsolutions/eip/route/orders/OdooProcessOrderRouteTest.java

```diff
@@ -864,6 +946,89 @@ public void shouldProcessQuantityDetailsAndDosingInstructionsForARevisionDrugOrd
+		mockPatientHandlerEndpoint.expectedPropertyReceived(EX_PROP_CREATE_CUSTOMER, true);
+		mockPatientHandlerEndpoint.whenAnyExchangeReceived(e -> e.setProperty(EX_PROP_ODOO_PATIENT_ID, odooPatientId));
+		
+		mockGetDraftQuotesEndpoint.expectedMessageCount(1);
+		mockGetDraftQuotesEndpoint.expectedPropertyReceived(EX_PROP_ODOO_PATIENT_ID, odooPatientId);
+		mockGetDraftQuotesEndpoint.whenAnyExchangeReceived(e -> e.getIn().setBody(new Map[] {}));
+		
+		mockManageQuoteEndpoint.expectedMessageCount(1);
+		mockManageQuoteEndpoint.expectedPropertyReceived(EX_PROP_ODOO_OP, ODOO_OP_CREATE);
+		final Integer quoteId = 9;
+		mockManageQuoteEndpoint.whenAnyExchangeReceived(e -> e.getIn().setBody(quoteId));
+		
+		final Integer qty = 2;
+		final String tabs = "Tabs";
+		final String qtyUnitsUuid = "some-units-uuid";
+		final Map quantityUnitsResource = new HashMap();
+		quantityUnitsResource.put("uuid", qtyUnitsUuid);
+		quantityUnitsResource.put("display", tabs);
+		final Double dose = 500.0;
+		final String mg = "mg";
+		final Map doseUnitsResource = singletonMap("display", mg);
+		final String daily = "Daily";
+		final Map frequencyResource = singletonMap("display", daily);
+		final Integer duration = 1;
+		final String week = "Week";
+		final Map durationUnitsResource = singletonMap("display", week);
+		orderResource.put("dose", dose);
+		orderResource.put("doseUnits", doseUnitsResource);
+		orderResource.put("frequency", frequencyResource);
+		orderResource.put("duration", duration);
+		orderResource.put("durationUnits", durationUnitsResource);
+		orderResource.put("quantity", qty);
+		orderResource.put("quantityUnits", quantityUnitsResource);
+		mockGetExtIdMapEndpoint.expectedMessageCount(1);
+		mockGetExtIdMapEndpoint.expectedPropertyReceived(EX_PROP_MODEL_NAME, "uom.uom");
+		mockGetExtIdMapEndpoint.expectedPropertyReceived(EX_PROP_EXT_ID, qtyUnitsUuid);
+		final Integer unitsId = 4;
+		mockGetExtIdMapEndpoint
+		        .whenAnyExchangeReceived(e -> e.getIn().setBody(new Map[] { singletonMap("res_id", unitsId) }));
+		
+		producerTemplate.send(URI_PROCESS_ORDER, exchange);
+		
+		mockManageQuoteEndpoint.assertIsSatisfied();
+		mockProcessNewOrderEndpoint.assertIsSatisfied();
+		mockProcessRevOrderEndpoint.assertIsSatisfied();
+		mockProcessDcOrVoidedOrderEndpoint.assertIsSatisfied();
+		mockGetExtIdMapEndpoint.assertIsSatisfied();
+		assertNull(exchange.getProperty(EX_PROP_ORDER_LINE));
+		assertEquals(0, exchange.getProperty(EX_PROP_ORDER_LINE_COUNT));
+		assertEquals(quoteId, exchange.getProperty(EX_PROP_QUOTE_ID));
+		assertEquals(unitsId, exchange.getProperty(EX_PROP_UNITS_ID));
+		final String description = drugName + " " + dose.toString() + mg + ", " + daily + ", " + duration + " " + week + " ("
+		        + qty + " " + tabs + "), " + "Orderer: " + ordererResource.get("display");
+		assertEquals(description, exchange.getProperty(EX_PROP_DESC));
+		System.out.println(description);
```

Comment:
Keeping System.out.println in the code?

Reply:
Thanks for the catch @wluyima, I've added a followup commit https://github.com/mekomsolutions/eip-routes-odoo/pull/5. Could it be merged in as well? Thanks.